### ISSUE TITLE
evaluation bug

### ISSUE THREAD

**Issue by qilinli** [BassyKuo/PSPNET-cudnn5#3]
Hi @BassyKuo,
 Thanks for proving cudnn version.
 I am using your branch for this PSPNet and trying to reproduce the result. The problem is I found there are a lot of code bugs in eval_all.m. Did you successfully reproduce the results, I mean those scores. I can get the segmented images as eval_sub.m is perfect, but I cannot get the scores. 

Cheers

**Comment by qilinli** [comment]
sorry not the eval_all.m, should be eval_acc.m.

**Comment by BassyKuo** [comment]
I did not test the evaluation code because there are some incompatible places that I have no time to rewrite in Octave (I don't have Matlab). But you can still provide the problem you encountered, we can fix it together :)

**Comment by qilinli** [comment]
@BassyKuo I think I found the reason of error. It comes from the PSPNET-cudnn5/evaluation/evaluationCode/intersectionAndUnion. Since I am testing the pre-trained model on Pascal voc 2012 test set, there is no groundtruth label (at least i did not found). So when it calls the function, the original jpg images are assigned to imLab which is w*d*3, and it doesn't match with imPred which is gray w*d. I guess the only to get mIou score for voc 2012 test is to submit to the evaluation server.

**Comment by BassyKuo** [comment]
@qilinli I think you are right. [eval_acc.m](https://github.com/BassyKuo/PSPNET-cudnn5/blob/master/evaluation/eval_acc.m#L36) needs the label ground truth to calculate the accuracy.
I will update this issue in the README file.
Thanks for your job!
